src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -1082,7 +1082,7 @@ public void closeChannel() {
       // telemetry and metrics
       this.telemetryServiceV2.reportKafkaPartitionUsage(this.snowflakeTelemetryChannelStatus, true);
       this.snowflakeTelemetryChannelStatus.tryUnregisterChannelJMXMetrics();
-    } catch (InterruptedException | ExecutionException e) {
+    } catch (InterruptedException | ExecutionException | SFException e) {
```

Comment:
Almost all the exceptions from the SDK will be a SFException, it's not ideal, but should we check the specific error code?

Reply:
Ideally we should add a closeWithoutVerification function in the SDK and use that instead

Reply:
After a discussion with @sfc-gh-japatel we've decided to catch all `SFException`s, as no matter what the problem is, we don't want to fail a connector task because of a channel.

Reply:
In general, it's an anti-pattern to catch all exceptions. This would just masks other unexpected exceptions and it's always better to fail faster IMO.

Reply:
I agree this is catch all exception but this is catch all only for closeChannel right? I think idea is to not have any failure when there is a shutdown since the channels are being opened in task open API anyways.